Packs/CommonScripts/Scripts/ServerLogs/ServerLogs.yml

```diff
@@ -16,6 +16,6 @@ dependson:
 tags:
 - widget
 type: python
-fromversion: 5.0.0
+fromversion: 6.0.0
```

Comment:
Didnt we decided on keeping the version?

Reply:
It doesn't appear to matter because we don't support 5.0.0.
For consistency, everything less than 6.0.0 will be bumped up to 6.0.0.